homeassistant/components/hunterdouglas_powerview/entity.py

```diff
@@ -31,7 +31,7 @@ def __init__(
         """Initialize the entity."""
         super().__init__(coordinator)
         self._room_name = room_name
-        self._attr_unique_id = unique_id
+        self._attr_unique_id = f"{device_info.serial_number}_{unique_id}"
```

Comment:
What is the unique_id in this context? Can anything be more unique than the serial number?

Reply:
every entity simply returns `unique_id` as an integer. Some of these were not throwing errors as we were using
`self._attr_unique_id = f"{self._attr_unique_id}_{description.key}"` which would resolve in something like `18_identify`

but the default returned by the hub is just an integer.

I went with prefixing with serial number to conform them to a standard.
Is there an issue using `device_info.serial_number` ? It was literally the first option under acceptable unique_id's so I didnt look further

Reply:
Oh no it is perfect, but having a variable `unique_id` kinda throws one off since you could see that as "unique" while in this context, it's not unique at all, it needs to be combined with something unique to be unique.

So renaming it to `unique_id_suffix` would make more sense as that describes the thing better